plugins-compat-tester/src/main/resources/resultToReport.xsl

```diff
@@ -265,7 +265,20 @@ th.version {
 	    <xsl:attribute name="id"><xsl:value-of select="$id" /></xsl:attribute>
 	    <xsl:attribute name="alt"><xsl:value-of select="$title" /></xsl:attribute>
 	    <xsl:attribute name="title"><xsl:value-of select="$title" /><xsl:if test="not($error='_')"><xsl:value-of select="$error" /></xsl:if></xsl:attribute>
-	    <xsl:attribute name="src">https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/<xsl:value-of select="$img" /></xsl:attribute>
+        <xsl:choose>
```

Comment:
The document is a mess in terms of tabs and spaces.  It appears there's a rogue tab in all the other lines throwing the doc out of alignment.  Fixing.